tests/dragonfly/snapshot_test.py

```diff
@@ -160,6 +160,26 @@ async def test_snapshot(self, df_seeder_factory, df_server):
         assert super().get_main_file("test-periodic-summary.dfs")
 
 
+@dfly_args({**BASIC_ARGS, "dbfilename": "test-periodic", "snapshot_cron": "*/2 * * * * *"})
```

Comment:
Can you comment on the meaning of the cron expression? (like "save every X seconds")

Reply:
It's set to save every 2 seconds.
You want me to add a comment in the code?

Reply:
yes please :)